qt/applications/workbench/workbench/plotting/figuremanager.py

```diff
@@ -82,11 +83,27 @@ def deleteHandle(self, _, workspace):
         all_axes = self.canvas.figure.axes
         if not all_axes:
             return
-        empty_axes = False
+
+        # Here we wish to remove any artists from a workspace that has been
```

Comment:
"remove any artists from the figure that are linked to a workspace that has been deleted"?